docs/integrations/airflow/extractors/custom-extractors.md

```diff
@@ -4,14 +4,11 @@ sidebar_position: 1
 
 # Custom extractors
 
-OpenLineage integration works by detecting which Airflow operators your dag is using, and extracting lineage 
-data from them using extractors. 
+OpenLineage integration works by detecting which Airflow operators your dag is using, and extracting lineage data from them using extractors. 
 
-However, there are hundreds of operators in Airflow. In addition, many people and teams write their own operators to 
-automate repeatable work - like using the same code from `PythonOperator` everywhere.
+However, not all operators are covered, in particular third party providers might not. To handle this situation, OpenLineage allows you to provide custom extractors for any operators.
```

Comment:
"However, not all operators are covered. In particular, third party providers may not be. To handle this situation, OpenLineage allows you to provide custom extractors for any operators where there is not one built-in"